### PR TITLE
Make commands handle spaces correctly

### LINKED ISSUE #88: Error on space in directory path
If there is a space in the directory path the compilation will fail.

Error log:
```
Error: Cannot find module 'C:\Dev'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.Module._load (module.js:276:25)
    at Function.Module.runMain (module.js:441:10)
    at startup (node.js:139:18)
    at node.js:968:3
```

### PR DESCRIPTION
**Pull request by nigredo-tori**
Fixes #88

### CODE REVIEW

File: sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala [hunk 1]
```diff
@@ -68,7 +68,7 @@ object ReloadWorkflow {
         }
       val concatFile = targetDir / "scalajsbundler-concat.js"
       IO.write(concatFile, concatContent.show)
-      Commands.run(s"node ${concatFile.absolutePath}", workingDir, logger)
+      Commands.run(Seq("node", "concatFile.absolutePath"), workingDir, logger)
```
Comment (julienrf) [inline sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala]: It should be `Seq("node", concatFile.absolutePath)` here :)

Comment (nigredo-tori) [inline sbt-scalajs-bundler/src/main/scala/scalajsbundler/ReloadWorkflow.scala]: Yeah, I've just found this myself. Sorry.

**Comment by julienrf** [comment]
test this please

**Comment by julienrf** [comment]
Thanks!

**Pull request merged**